### PR TITLE
fixes #294 - refactored delete-signature to manage-signatures

### LINKED ISSUE #294: Document signature operation like list and inspect a signature
In general, users can view a signature associated with an image by using `notation list` and get the details of signature information by using `notation inspect`. There is a [document ](https://notaryproject.dev/docs/how-to/delete-signature/) to explain how to delete a signature. I am thinking if we could revise it to the topic **signature operation** and add other signature operation commands like `notation list` and  `notation inspect` to [this document ](https://notaryproject.dev/docs/how-to/delete-signature/). 

cc @zr-msft

### PR DESCRIPTION
**Pull request by zr-msft**
fixes #294

### CODE REVIEW

File: content/en/docs/how-to/manage-signatures.md [hunk 1]
```diff
@@ -0,0 +1,119 @@
...
+{{% /alert %}}
+
+## List signatures on an artifact
+
+Use `notation list` to show the signatures associated with an artifact.
+
+{{% alert title="Important" color="important" %}}
+When using `notation list`, use the fully qualified artifact reference, including the digest. For example, `localhost:5001/net-monitor@sha256:111222333444555666777888999000aaabbbcccdddeeefff0001112223334445`. You can use `notation list` with a tag, for example, `localhost:5001/net-monitor:latest`. Using a tag is not recommended since a tag is mutable and may be updated to point to a different version of the artifact. A digest is not mutable and always points to the same version of the artifact.
+
+To find the digest of an artifact, use `notation list` with the artifact reference. For example, `notation list localhost:5001/net-monitor:v1`. You can also use `oras manifest fetch --descriptor` on remote registries that do not require authentication. For example, `oras manifest fetch --descriptor REGISTRY/IMAGE:TAG`.
```
Comment (FeynmanZhou) [inline content/en/docs/how-to/manage-signatures.md:19]:
```suggestion
To find the digest of an artifact, use `notation list` with the artifact reference. For example, `notation list localhost:5001/net-monitor:v1`. You can also use `oras manifest fetch --descriptor` on OCI registries. For example, `oras manifest fetch --descriptor REGISTRY/IMAGE:TAG`.
```

Comment (FeynmanZhou) [reply content/en/docs/how-to/manage-signatures.md:19]: ORAS supports all OCI-compatible registries but it also requires to authenticate with the registries before using ORAS.

**Review by FeynmanZhou — COMMENTED** [review]
@zr-msft  PTAL, thanks!

File: content/en/docs/how-to/manage-signatures.md (hunk 1 above)
Comment (zr-msft) [reply content/en/docs/how-to/manage-signatures.md]: thanks @FeynmanZhou 

fixed

**Review by FeynmanZhou — APPROVED** [review]
LGTM

**Review by toddysm — APPROVED** [review]

**Pull request merged**